release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -772,6 +772,15 @@ If you customized the default OpenSSH `/etc/ssh/sshd_config` server configuratio
 
 The {cert-manager-operator} 1.11 is now generally available in {product-title} 4.14 as well as {product-title} 4.13 and {product-title} 4.12.
 
+[discrete]
+[id="ocp-4-14-ovn-k-interconnect"]
+=== OVN-Kuberenetes interconnect
+{product-title} {product-version} introduces OVN-Kubernetes interconnect architecture to {openshift-networking}. This techincal change enables scaling clusters linearly with the number of nodes with a perfomance that is similar to OpenShift SDN.
+
+The new architecture also removes external Transport Layer Security (TLS) over proxy connection requirement from each hosted cluster to management cluster, which fulfills hosted control plane requirements. Users can now place more hosted cluster control planes on the management cluster.
+
+This update localizes databases to a specific node so that if a database goes down only that node is affected. See xref:../networking/ovn_kubernetes_network_provider/ovn-kubernetes-architecture-assembly.adoc#ovn-kubernetes-architecture-assembly[OVN-Kubernetes architecture] for more on the interconnect architecture.
```

Comment:
```suggestion
This update localizes databases to a specific node so that if a database goes down, only that node is affected. See xref:../networking/ovn_kubernetes_network_provider/ovn-kubernetes-architecture-assembly.adoc#ovn-kubernetes-architecture-assembly[OVN-Kubernetes architecture] for more on the interconnect architecture.
```

Reply:
please mention you are talking about OVN databases and mention it went from being centralized to distributed per node.